server/app.py

```diff
@@ -25,12 +25,13 @@
 from aiortc.rtcrtpsender import RTCRtpSender
 from aiortc.codecs import h264
 from pipeline import Pipeline
-from utils import patch_loop_datagram, StreamStats, add_prefix_to_app_routes
+from utils import patch_loop_datagram, add_prefix_to_app_routes
+from metrics import metrics_handler, StreamStats, update_metrics, enable_metrics
 import time
 
 logger = logging.getLogger(__name__)
-logging.getLogger('aiortc.rtcrtpsender').setLevel(logging.WARNING)
-logging.getLogger('aiortc.rtcrtpreceiver').setLevel(logging.WARNING)
+logging.getLogger("aiortc.rtcrtpsender").setLevel(logging.WARNING)
```

Comment:
@eliteprox Sorry for these changes. I can get them out if you want. I have black formatting running on my system. We might want to set up pre-commit hooks to ensure that formatting is applied to all commits.